packages/oui-navbar/README.md

```diff
@@ -14,31 +14,249 @@ oui-navbar is a package which provide styles for the navbar component.
 
 ```html:preview
```

Comment:
Change to html only (maybe temporarly)

Reply:
The doc will be changed with simpler examples :)